src/main/treeDataProviders/utils/descriptorUtils.ts

```diff
@@ -20,6 +20,11 @@ import { FocusType } from '../../constants/contextKeys';
 import { DescriptorIssuesData } from '../../types/issuesData';
 
 export class DescriptorUtils {
+    public static readonly DESCRIPTOR_SELECTOR: vscode.DocumentSelector = {
+        scheme: 'file',
+        pattern: '**/{go.mod,package.json,pom.xml,*requirements*.txt,*.sln,yarn.lock}'
```

Comment:
Why is the `*sln`? NuGet is not supported in getDependencyPosition

Reply:
removed